src/main/java/core/basesyntax/strategy/OperationStrategy.java

```diff
@@ -0,0 +1,26 @@
+package core.basesyntax.strategy;
+
+import core.basesyntax.operation.Operation;
+import core.basesyntax.service.OperationService;
+import core.basesyntax.service.impl.BalanceOperation;
+import core.basesyntax.service.impl.PurchaseOperation;
+import core.basesyntax.service.impl.ReturnOperation;
+import core.basesyntax.service.impl.SupplyOperation;
+
+public class OperationStrategy {
+    public static OperationService getOperation(Operation operation) {
+        if (operation.getCode().equals(Operation.BALANCE.getCode())) {
+            return new BalanceOperation();
+        }
+        if (operation.getCode().equals(Operation.SUPPLY.getCode())) {
+            return new SupplyOperation();
+        }
+        if (operation.getCode().equals(Operation.PURCHASE.getCode())) {
+            return new PurchaseOperation();
+        }
+        if (operation.getCode().equals(Operation.RETURN.getCode())) {
+            return new ReturnOperation();
+        }
+        throw new RuntimeException("This operation doesn't exist " + operation);
+    }
```

Comment:
1) Make method non-static. It may look easier now but static elements are much harder to be tested and maintained properly
2) It's strange when method that accepts Operation is called getOperation. With such naming I expect to see single line of code ```return operation``` :)  Should be getOperationHandler() or anything that indicates that we will obtain special component that is ready to perform some logic using our operation
3) Please use map instead of ```if``` chain